app/models/user.rb

```diff
@@ -554,6 +554,13 @@ def reactions_to
     Reaction.for_user(self)
   end
 
+  def last_activity
+    return unless registered == true
+
+    [last_comment_at, last_article_at, latest_article_updated_at, last_reacted_at, profile_updated_at,
```

Comment:
If this resolves to nil then the `format_last_activity_timestamp` will essentially error out because timestamp will be nil. 

Hence perhaps we should use the registered date in this array as a fallback for last activity.